superset/config.py

```diff
@@ -244,7 +244,7 @@ def _try_json_readsha(filepath: str, length: int) -> str | None:
 ]
 
 # Whether to run the web server in debug mode or not
-DEBUG = os.environ.get("FLASK_ENV") == "development"
+DEBUG = os.environ.get("FLASK_DEBUG")
```

Comment:
Should there be logic to convert a truthy environment variable to a Python bool?

Reply:
Good catch, will update.